README.md

```diff
@@ -1,2 +1,169 @@
-# xchainClient
-The xchain clients takes care of data bridge events
+# Xchain Client
+
+Xchain Client provides a set of tools to facilitate cross-chain data movement from Filecoin storage to any blockchain. It includes utilities for managing Ethereum accounts, submitting offers, and handling deal status.
+
+---
+
+## 🚀 Installation
+
+Ensure you have **Go 1.18+** installed. Then, clone the repository and build the project:
+
+```sh
+git clone https://github.com/your-repo/xchainClient.git
+cd xchainClient
+go build -o xchainClient
+```
+---
+
+## 📌 Configure Environment Variables
+1. Create a `.env` file to store your **XCHAIN_PASSPHRASE** (used for unlocking the Ethereum keystore):
```

Comment:
Is this XCHAIN_PASSPHRASE a new password that will be used to create a new Ethereum keystore/wallet? I might be confused and not sure where to get this password.